docs/doxygen-dev/regression_testing.dox

```diff
@@ -0,0 +1,217 @@
+/*! \page regression_testing Regression Testing in Autopsy
+
+\section intro_regression Introduction
+The autopsy/test folder contains scripts that are necessary to run the regression test. Developers that are interested in running regression tests should run autopsy/test/script/regression.py. This guide will familiarize you with regression.py and walk you through a simple example at the end.
+
+If you’ve never heard of regression testing, see <a href="https://en.wikipedia.org/wiki/Regression_testing">Regression Testing</a>.
+
+From a bird’s eye view, regression.py will:
+- Run ingest on an image by starting up Autopsy and controlling the UI
+- Generate a HTML report once ingest is complete
+- Dump the case database and perform diffs on the gold standards. HTML reports are also diffed.
```

Comment:
You could use an anchor on your terms to link directly to them. For example, put '\anchor gold_standard' in front of the gold standard definition, and then here use '\ref gold_standard "gold standards"'